src/state/slices/opportunitiesSlice/opportunitiesSlice.ts

```diff
@@ -1,6 +1,11 @@
 import { createSlice } from '@reduxjs/toolkit'
+import type { BaseQueryError } from '@reduxjs/toolkit/dist/query/baseQueryTypes'
 import { createApi } from '@reduxjs/toolkit/query/react'
 import { DefiProvider } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import * as E from 'fp-ts/lib/Either'
+import { pipe } from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import * as TE from 'fp-ts/lib/TaskEither'
```

Comment:
I thought about the best way to handle this, and this "middle ground" namespace import approach is probably just right 👌